packages/trpc/server/routers/viewer/apps.tsx

```diff
@@ -142,6 +142,8 @@ export const appsRouter = router({
 
       // If disabling an app then we need to alert users based on the app type
       if (input.enabled) {
+        const translations = new Map(["en", await getTranslation("en", "common")]);
```

Comment:
English will always be loaded, even when the user does not use it?

Reply:
Updated with the changes.